misc/lldb_rb/rb_base_command.py

```diff
@@ -67,3 +67,191 @@ def _append_command_output(self, debugger, command, result):
         result.Clear()
         result.write(output1)
         result.write(output2)
+
+    def lldb_inspect(debugger, target, result, val):
```

Comment:
Is there some way to get rid of the duplicated `lldb_inspect` in `lldb_cruby.py` and use this instead?

Reply:
I'm not super concerned about this at the moment. Eventually there shouldn't be anything left in this file besides the object hierarchy initialisation when everything gets ported across anyway.